root/statistics/utilities.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @flow
+ * Copyright (C) 2018 Shamroy Pellew
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+const serializeObject = (obj) => {
+  return Object.keys(obj).map(key => key + '=' + encodeURIComponent(obj[key])).join('&');
+};
+
+exports.addColon = (str: string) => {
+  return str + ':';
+};
+
+exports.formatPercentage = (num: number, digits: number, $c: CatalystContextT) => {
```

Comment:
Use `export function formatPercentage(...) {}` instead of `exports.formatPercentage` (mixing ES module and CommonJS syntax in the same file will cause problems later, particularly with Webpack).